config/application.rb

```diff
@@ -1,7 +1,5 @@
 # frozen_string_literal: true
 
-require_relative 'boot'
```

Comment:
I'm pretty sure we need this line: https://github.com/rails/rails/blob/master/railties/lib/rails/generators/rails/app/templates/config/application.rb.tt#L1

Reply:
i was also surprised by the disappearance of that line, i'm pretty sure this came from the `rails app:update` invocation, but also could've been bad merging on my part.  will add back...

Reply:
It is there in a newly generated rails 6.1, so presumably we need it.